pkg/types/powervs/powervs_regions.go

```diff
@@ -65,3 +69,15 @@ var Regions = map[string]Region{
 		Zones:       []string{"us-east"},
 	},
 }
+
+// VPCRegionForRegion returns the VPC region for the specified region.
+func VPCRegionForRegion(region string) (string, error) {
+	var r Region
+	var ok bool
+
+	if r, ok = Regions[region]; ok {
+		return r.VPCRegion, nil
+	}
```

Comment:
```suggestion

	if r, ok := Regions[region]; ok {
		return r.VPCRegion, nil
	}
```

Reply:
Talk about an extreme level of nit picking!

Reply:
this is what upstream will ask for as well. thanks @Karthik-K-N :)

Reply:
I would argue that this obfuscation removes helpful information for future developers as I have spent time in the past figuring out the correct type.  But I guess we don't care about being helpful.